settings/docs/getting-started.md

```diff
@@ -0,0 +1,272 @@
+---
+title: "Getting Started"
+date: 2018-05-02T00:00:00+00:00
+weight: 25
+geekdocRepo: https://github.com/owncloud/ocis-settings
+geekdocEditPath: edit/master/docs
+geekdocFilePath: getting-started.md
+---
+
+{{< toc >}}
+
+## Installation
+
+So far we are offering two different variants for the installation. You can choose between [Docker](https://www.docker.com/) or pre-built binaries which are stored on our download mirrors and GitHub releases. Maybe we will also provide system packages for the major distributions later if we see the need for it.
+
+### Docker
+
+TBD
+
+### Binaries
+
+TBD
+
+## Configuration
+
+We provide overall three different variants of configuration. The variant based on environment variables and commandline flags are split up into global values and command-specific values.
+
+### Envrionment variables
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)